api/drop_chunks.md

```diff
@@ -33,14 +33,16 @@ based on a hash partition.
 |Name|Type|Description|
 |-|-|-|
 |`relation`|REGCLASS|Hypertable or continuous aggregate from which to drop chunks.|
-|`older_than`|INTERVAL|Specification of cut-off point where any full chunks older than this timestamp should be removed.|
 
 ## Optional arguments
 
 |Name|Type|Description|
 |-|-|-|
-|`newer_than`|INTERVAL|Specification of cut-off point where any full chunks newer than this timestamp should be removed.|
+|`older_than`|ANY|Specification of cut-off point where any full chunks older than this timestamp should be removed.|
+|`newer_than`|ANY|Specification of cut-off point where any full chunks newer than this timestamp should be removed.|
 |`verbose`|BOOLEAN|Setting to true displays messages about the progress of the reorder command. Defaults to false.|
+|`created_before`|ANY|Specification of cut-off point where any full chunks created before this timestamp should be removed.|
+|`created_after`|ANY|Specification of cut-off point where any full chunks created after this timestamp should be removed.|
```

Comment:
Slightly hard to read. See suggestions above.

Reply:
same reasoning as above. It can be a range when both are specified.